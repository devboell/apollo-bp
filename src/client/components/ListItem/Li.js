import styled from 'styled-components';

const Li = styled.li`
  color: rgb(222, 81, 11);
`;

export default Li;
