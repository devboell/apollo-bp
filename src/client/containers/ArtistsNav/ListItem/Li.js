import styled from 'styled-components'

const Li = styled.li`
  > a {
    text-decoration: none;
  }
`;

export default Li
