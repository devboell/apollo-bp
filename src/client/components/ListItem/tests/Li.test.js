import React from 'react'
import renderer from 'react-test-renderer'
import Li from '../Li'

it('Li component snapshot', () => {
  const tree = renderer.create(<Li/>).toJSON()
  expect(tree).toMatchSnapshot()
});
