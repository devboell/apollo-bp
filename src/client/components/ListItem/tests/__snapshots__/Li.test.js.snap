// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Li component snapshot 1`] = `
<li
  className="bfXghp"
/>
`;
